user/tests/wiring/adc_dac/adc_dac.cpp

```diff
@@ -71,3 +72,20 @@ test(ADC_AnalogReadOnPinWithDACOutputResultsInCorrectValue) {
     //To Do : Add test for remaining pins if required
 }
 #endif
+
+#if (PLATFORM_ID == 6)
```

Comment:
This should be `#if PLATFORM_ID >= 3` to make sure this runs properly on P1 and Electron as well.  Both PLATFORM_ID==6 in this file could be changed to this.  Also what does the for() loop test since the pin is set back to `analogWrite(DAC, 2048);` afterwards?  Also looking at the setup, it seems A5 is biased to Vdd/2, which is what the DAC is set to as well.  I would think we would want to bias the pin to maybe 1/3Vdd and then set the DAC to 2/3Vdd for a proper test.  As-is the DAC could be not outputting anything at HI-Z and the test would pass.  100k ohm resistors might be too high as well for a proper ADC reading.  Sorry for all the grief I know you did not write the initial test, just documenting some comments about it now that I'm actually looking at it for the first time ;)